subsys/bluetooth/host/Kconfig

```diff
@@ -311,6 +311,14 @@ config BT_BONDABLE
 	  Bonding flag in AuthReq of SMP Pairing Request/Response will be set
 	  indicating the support for this mode.
 
+config BT_OOB_DATA_FIXED
+	bool "Use a fixed random number for LESC OOB pairing"
```

Comment:
It might be wise to add a `depends on BT_TESTING` for this one, to reduce risk of someone enabling it for a production build.